src/index.tsx

```diff
@@ -4,6 +4,10 @@ import './index.css'
 import App from './App'
 
 const urlParams = new URLSearchParams(window.location.search)
-const path = urlParams.get('path') || '/data/scenarios/default'
+const path = (
+    urlParams.get('path') ||
+    process?.env?.REACT_APP_SFF_DEFAULT_SCENARIO ||
```

Comment:
Nice one, this will allow us to load scenarios from any URL by default

Reply:
It also enables the use of `.env.local` in order to set a specific default scenario during development without risking unintended changes to the system. :)

Reply:
Really nice!